pyeti/eti_django/store/admin.py

```diff
@@ -88,3 +90,20 @@ def sync_from_store(self, request, queryset):
         if failure:
             self.message_user(request, '%s license(s) failed to sync from store' % failure, level=messages.ERROR)
     sync_from_store.short_description = 'Sync from store'
+
+    def export_as_csv(self, request, queryset):
+
+        meta = self.model._meta
+        field_names = [field.name for field in meta.fields]
```

Comment:
@theunraveler so would you update `field_names` to be `field_names = ['token', 'num_seats', 'start_date', 'end_date', 'spree_order_number']` ?

Reply:
Yeah, I'd try interacting with the admin class to get those fields automatically. Sec and I'll send some code.

Reply:
Well, nevermind--involving the admin class to render the field values is a little harder than I thought. So yeah, I'd just update the `field_names` like you suggested.